apps/site/mix.exs

```diff
@@ -127,7 +128,8 @@ defmodule Site.Mixfile do
       {:trip_plan, in_umbrella: true},
       {:services, in_umbrella: true},
       {:route_patterns, in_umbrella: true},
-      {:repo_cache, in_umbrella: true}
+      {:repo_cache, in_umbrella: true},
+      {:con_cache, "~> 0.12.0"}
```

Comment:
Currently these seem to be organized as "non-umbrella deps" followed by "umbrella deps"; could we put this one up there with the others of its kind?